docs/workforce-management-framework/topics/pro-keycloak-enablement.adoc

```diff
@@ -1,15 +1,19 @@
 [id='{context}-pro-keycloak-enablement']
 = Enabling {Keycloak}
 
-The xref:{context}-adding-keycloak-configuration[Adding {Keycloak} Configuration] and xref:{context}-enabling-keycloak-on-the-demo-server[Enabling {Keycloak} on the Demo Server] headings below are valid for any instance of *{Keycloak}* provided you implement the _Realm settings_.
-
 IMPORTANT: Enabling *{Keycloak}* requires the disabling of *PassportAuth*.
 
 [discrete]
 == Prerequisites
 
 . Ensure you have completed xref:getting-started[Getting Started]
 . Docker (tested with version: {WFM-RC-DockerVersion})
+. Knowledge of {Keycloak} terminologies. See xref:{context}-understanding-keycloak-terminologies[Understanding {Keycloak} Terminologies] section for more information.
+
+NOTE: If you choose to run your own {Keycloak} instance, ensure that a realm is setup for {PROJECT_NAME} applications
```

Comment:
I'm not expert here but this note may be really long and it's actually separate use case/heading. 
Leaving that for @jstaffor to decide.

Reply:
There is too much info in the note for the heading '== Prerequisites IMO'. I would keep the note to -->

 - NOTE: If you use your own {Keycloak} instance, ensure that a realm is setup for {PROJECT_NAME} applications to use. 

The rest of the info, can it be integrated at a logical location later on in this text? The only suggestion I can think of is if you break the heading "Importing the {Keycloak} Realm Settings" into 2 section like so:

= Importing the {Keycloak} Realm Settings

== Importing the default {Keycloak} Realm Settings
<Add info under the heading ' Importing the {Keycloak} Realm Settings' to here>
== Importing custom {Keycloak} Realm Settings
There are two options to setup the realm:

. Import the existing demo link:{WFM-RC-KeycloakDockerURL}{WFM-RC-Branch}/data_files/raincatcher-realm.json[{PROJECT_NAME} realm settings] (see xref:{context}-import-keycloak-realm-settings[Importing the {Keycloak} Realm Settings] section for more information)
. Create your own realm that can be used with the {PROJECT_NAME} applications (see xref:{context}-create-own-keycloak-realm[Creating Your own {Keycloak} Realm] section for more information).

Reply:
@JameelB Agree with @jstaffor here and in couple places we building some large `NOTE` sections. If we try to extract them a little.